src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3091,25 +3097,25 @@ QString ViewProviderSketch::iconTypeFromConstraint(Constraint *constraint)
     /*! TODO: Consider pushing this functionality up into Constraint */
     switch(constraint->Type) {
     case Horizontal:
-        return QString::fromLatin1("small/Constraint_Horizontal_sm");

```

Comment:
Is this the only place small SVGs are used? Delete then?

Reply:
May be. Check three times before deleting.